examples/integration_test.py

```diff
@@ -251,8 +251,10 @@ def verify_avro():
     prim_float = avro.load(os.path.join(avsc_dir, "primitive_float.avsc"))
     prim_string = avro.load(os.path.join(avsc_dir, "primitive_string.avsc"))
     basic = avro.load(os.path.join(avsc_dir, "basic_schema.avsc"))
-    str_value = 'abc'
```

Comment:
Don't delete or change existing test cases (unless they are wrong), just add new ones, otherwise we loose test coverage and regression testing.